styles.css

```diff
@@ -0,0 +1,734 @@
+* {
+  font-family: 'Cocogoose', sans-serif;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+.cursor {
+  cursor: pointer;
+}
+
+.picture {
+  width: 100px;
+}
+
+h1 {
+  font-size: 30px;
+  font-weight: 700;
+  color: #ec5242;
+}
+
+.red {
+  color: #ec5242;
+}
+
+.gray {
+  color: #d3d3d3;
+}
+
+.black {
+  color: #272a31;
+}
+
+.blackbg {
+  background-color: #272a31;
+}
+
+.whitebg {
+  background-color: white;
+}
+
+.b {
+  border: 2px solid red;
+}
+
+/* bodi {
+    background: url("media/background.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+
+}
+  */
+body {
+  background: url("media/background.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+@media (max-width: 768px) {
+  .window {
+    display: block;
```

Comment:
- [ ] In your project you are using desktop first approach to style, but according to requirements  you should use mobile-first approach: 

<img width="958" alt="Screenshot 2022-03-11 at 18 42 30" src="https://user-images.githubusercontent.com/77038610/157889217-88fbe2c0-6b9d-4264-987e-45c8362b2f2e.png">

This is how media queries are written for mobile-first approach : 

<img width="288" alt="Screenshot 2022-03-11 at 18 43 40" src="https://user-images.githubusercontent.com/77038610/157889442-27c6eeed-eab5-4ee7-bbbf-9a103ca19503.png">